website/src/components/ui/common/TranslationStatusAlert.tsx

```diff
@@ -43,6 +43,16 @@ const getStatusConfig = (status: TranslationStatus): StatusConfig => {
 				message:
 					"このページはまだ翻訳されていません。原文の内容が表示されています。",
 			};
+		case "original":
+			return {
+				bgColor: "bg-cyan-50",
+				borderColor: "border-cyan-200",
+				textColor: "text-cyan-800",
+				iconColor: "text-cyan-600",
+				label: "日本語版オリジナル",
+				message:
+					"このページの内容は公式ドキュメントには含まれておらず、日本語コミュニティが独自に追加したものです。",
```

Comment:
Typst Japan Communityであるため日本コミュニティがよいかと思います。それか`Typst Japan Community`とそのまま書いてしまってもよいかもしれません。

参考： https://github.com/typst-jp

```suggestion
					"このページの内容は公式ドキュメントには含まれておらず、Typst Japan Communityが独自に追加したものです。",
```

Reply:
これはOrganizationの命名そのものに対する懸念ですが、@typst-jp の実際の活動内容や対象は「日本語話者」であり、誤解や排他性を避けて多様な日本語話者を歓迎する姿勢を示すためにも、国ではなく言語話者として表記する方が多様性や国際性の観点から適切だと考えています。
ただし、TLDとの兼ね合いから、実態が日本語話者向けであっても`-jp`という命名を用いることがOSSや技術コミュニティの慣習として十分に認知されているため、identificationとしての`jp`は許容し、文章としては「日本語」や「Japanese」と明記するのが妥当だと思います。